manifests/node-maintenance-operator/node-maintenance-operator.package.yaml

```diff
@@ -1,4 +1,4 @@
 packageName: node-maintenance-operator
 channels:
 - name: beta
-  currentCSV: node-maintenance-operator.v0.4.0
+  currentCSV: node-maintenance-operator.v0.5.0
```

Comment:
Was this necessary?

Reply:
yes, we introduced a major change by switching package names; so I had to increment the CSV version.